StellarKit/source/Stellar.swift

```diff
@@ -165,7 +165,32 @@ public enum Stellar {
         
         return p
     }
-    
+
+    /**
+     Burn an account.
+
+     - Parameter balance: The given `Account`s current balance.
+     - Parameter asset: The `Asset` to trust.
```

Comment:
I would document this as the asset to burn.  That a CHANGE_TRUST op is used shouldn't matter to the caller.

Reply:
i just noticed this also. copy past!